File: DATOS ABSTRACTOS/DATOS ABS.cs
using System;
class Persona {
    public string nombre;
    public int edad;


    
    public Persona(string nombre, int edad) {
        this.nombre = nombre;
        this.edad = edad;
    }


    public string Saludar() {
        return $"Hola, soy {nombre} y tengo {edad} años.";
    }
}


// Uso del DATO abs
class Program
{
    static void Main()
    {
        Persona p1 = new Persona("Gustavo", 20);
        Console.WriteLine(p1.Saludar());
    }
}

File: DATOS ABSTRACTOS/DATOS ABS.cpp
#include <iostream>
using namespace std;


class Persona {
private:
    string nombre;
    int edad;


public:
    
    Persona(string n, int e) {
        nombre = n;
        edad = e;
    }


    
    string saludar() {
        return "Hola, soy " + nombre + " y tengo " + to_string(edad) + " años.";
    }
};


// Uso del Daato abs
int main() {
    Persona p1("Gustavo", 20);
    cout << p1.saludar() << endl;
    return 0;
}
